magento/magento2
# [MFTF] Refactoring of AdminMassProductPriceUpdateTest

### Description (*)
The test is refactored according to the best practices followed by MFTF.

### Manual testing scenarios (*)
1. Create 2 simple products
2. Go to Admin->Catalog->Products
3. Update price of the products via mass update
4. Check if price is updated on the Product Edit page (for both products)


### Resolved issues:
1. [x] resolves magento/magento2#31073: [MFTF] Refactoring of AdminMassProductPriceUpdateTest
